api/v1alpha1/customer_types.go

```diff
@@ -117,3 +125,15 @@ func (customer *Customer) IsValidUpdate(existingCustomer Customer) (bool, error)
 
 	return true, nil
 }
+
+func (customer *Customer) IsValidCustomerUpdate(existingCustomer Customer) (bool, error) {
+	if !strings.EqualFold(customer.Spec.Email, existingCustomer.Spec.Email) {
+		// Customer creation takes few seconds to complete. Checking if the customer is Get during the creation process.
+		if existingCustomer.Spec.Name == "User " && existingCustomer.Spec.Email == "?" {
+			return true, nil
+		}
```

Comment:
It takes a few seconds for customers to be persisted at JSD. Check if it's pending with known values are return.
```suggestion
		// It takes a few seconds for customers to be persisted at JSD. Check if it's pending with known values are return.
		if existingCustomer.Spec.Name == "User " && existingCustomer.Spec.Email == "?" {
			return true, nil
		}
```